variables.tf

```diff
@@ -1,100 +1,89 @@
 variable "fqdn" {
-  type        = string
+  type = string
   description = "The FQDN of the website and also name of the S3 bucket"
 }
 
 variable "aliases" {
-  type        = list(string)
+  type = list(string)
   description = "Any other domain aliases to add to the CloudFront distribution"
-  default     = []
+  default = []
 }
 
 variable "acceleration_status" {
-  type        = string
-  description = "The acceleration_status argument of the S3 bucket (warning: this is not possible right now for hosted sites, as bucket names cannot contain a dot.)"
-  default     = "Suspended"
+  type = string
+  description = "(Suspended | Enabled)The acceleration_status argument of the S3 bucket (warning: this is not possible right now for hosted sites, as bucket names cannot contain a dot.)"
+  default = "Suspended"
```

Comment:
@phuonghuynh this doesn't work, the "bucket names with a dot" will fail here even if acceleration_status is set to "Suspended". You may have to set default to `null`.

Reply:
I renamed `bucket_name` to use "-" ; https://github.com/riboseinc/terraform-aws-s3-cloudfront-website/pull/32/files#diff-beadbc29548607cf77c3c6882a0c49559eb088a560805d12a445c52f80f07b31R2

Reply:
I see, but this change will be incompatible with the old users, they will have to remove and re-create the bucket...

I'm hoping to only affect those people who want to use transfer acceleration.

Reply:
I will add support for "null" value for the status

Reply:
Exposing new variable "bucketName" to allow user control this.

Reply:
Got it. Can you also update documentation? Thanks.